x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -148,46 +193,35 @@ export const signalRulesAlertType = ({
             pageSize: searchAfterSize,
             tags,
           });
+        }
 
-          if (bulkIndexResult) {
-            logger.debug(
-              `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}"`
-            );
-            await writeCurrentStatusSucceeded({
-              services,
-              currentStatusSavedObject,
-            });
-          } else {
-            await writeSignalRuleExceptionToSavedObject({
-              name,
-              alertId,
-              currentStatusSavedObject,
-              logger,
-              message: `Bulk Indexing signals failed. Check logs for further details \nRule name: "${name}"\nid: "${alertId}"\nrule_id: "${ruleId}"\n`,
-              services,
-              ruleStatusSavedObjects,
-              ruleId: ruleId ?? '(unknown rule id)',
-            });
-          }
-        } catch (err) {
+        if (bulkIndexResult) {
+          logger.debug(
+            `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}"`
```

Comment:
I think this debugger line was probably always missing index like so:

```ts
          logger.debug(
            `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}", output_index: "${outputIndex}"`
          );
```

would be nice to just add it now so in debug mode people can track which index things are being written into.